pkgs/top-level/python-packages.nix

```
@@ -4929,7 +4929,7 @@ in {
 
   rply = callPackage ../development/python-modules/rply {};
 
-  rpm = toPythonModule (pkgs.rpm.override{inherit python;});
+  rpm = disabledIf (!isPy3k) (toPythonModule (pkgs.rpm.override{ inherit python; }));
```

The override should be unnecessary now that it will be disabled otherwise
```suggestion
  rpm = disabledIf (!isPy3k) (toPythonModule pkgs.rpm);
```

---

If the override is removed, it always build with the default python3 version in nixpkgs. I.e. `nix-build -A python36Packages.rpm` and `nix-build -A python37Packages.rpm` would be the same derivation, when they shouldn't be.

---

Didn't know about `isPy2`, thanks, added it to `/top-level/python-packages.nix`

---

oh right, there's no callPackage, LGTM then